ext4: Show delalloc options

From: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>

Delayed allocation is enabled by default.
So if they are are not set in mount options
that means the user explicity disabled them
using nomablloc and nodelalloc option. Show
the same in ext4_show_options

Signed-off-by: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>
Acked-by: Eric Sandeen <sandeen@redhat.com>
Signed-off-by: Mingming Cao <cmm@us.ibm.com>
Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>

---

 fs/ext4/super.c |    3 +++
 1 file changed, 3 insertions(+)


Index: linux-2.6.26-rc5/fs/ext4/super.c
===================================================================
--- linux-2.6.26-rc5.orig/fs/ext4/super.c	2008-06-06 17:04:00.000000000 -0700
+++ linux-2.6.26-rc5/fs/ext4/super.c	2008-06-06 17:04:01.000000000 -0700
@@ -753,6 +753,9 @@ static int ext4_show_options(struct seq_
 		seq_puts(seq, ",nomballoc");
 	if (test_opt(sb, I_VERSION))
 		seq_puts(seq, ",i_version");
+	if (!test_opt(sb, DELALLOC))
+		seq_puts(seq, ",nodelalloc");
+
 
 	if (sbi->s_stripe)
 		seq_printf(seq, ",stripe=%lu", sbi->s_stripe);
